wp-cli.host.yml

```diff
@@ -0,0 +1,2 @@
+ssh: vagrant:default
+path: /chassis/wordpress
```

Comment:
I think this should be `/wordpress` since we're pulling in our own copy of WordPress separate from Chassis'

Reply:
This is actually the path inside the virtual machine, so I think `/chassis/wordpress` is correct. (The core checkout from Chassis is at `/chassis/chassis/wp`.)

Reply:
Ah, of course 🤦‍♂️